integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioDomainInPV.java

```diff
@@ -255,6 +259,10 @@ public void testIstioDomainHomeInPv() {
                 .clusterName(clusterName)
                 .replicas(replicaCount)
                 .serverStartState("RUNNING"))
+            .addManagedServersItem(new ManagedServer()
+                .serverName("managed-server2")
```

Comment:
Replace with constant.

Reply:
Managed Server need not be added, as I am doing scale down/up in the absence of Admin Server.